connectors/src/connectors/slack/chat/stream_conversation_handler.ts

```diff
@@ -245,6 +255,27 @@ export async function streamConversationToSlack(
           },
           { adhereToRateLimit: false }
         );
+        if (
+          slackUserId &&
+          !slackUserInfo.is_bot &&
+          agentConfigurations.length > 0
```

Comment:
Can we filter out external users as well?

Reply:
As external user can't ask the assistant he won't get there - they will be filtered when calling `notifyIfSlackUserIsNotAllowed`

Reply:
👍